activesupport/lib/active_support/notifications.rb

```diff
@@ -209,11 +230,17 @@ def instrument(name, payload = {})
       #     @event = event
       #   end
       def subscribe(*args, &block)
-        notifier.subscribe(*args, &block)
+        pattern, callback = *args
+        notifier.subscribe(pattern, callback, false, &block)
+      end
+
+      def monotonic_subscribe(*args, &block)
+        pattern, callback = *args
+        notifier.subscribe(pattern, callback, true, &block)
       end
 
-      def subscribed(callback, *args, &block)
-        subscriber = subscribe(*args, &callback)
+      def subscribed(callback, pattern, monotonic = false, &block)
```

Comment:
can we use `monotonic: false` here?

Reply:
@rafaelfranca I have incorporated this feedback. Please review. 

Now the build on CI appears to be failing due to unrelated failures. I do not have permissions to rebuild it.